lib/dbservice_web/controllers/group_controller.ex

```diff
@@ -16,15 +18,38 @@ defmodule DbserviceWeb.GroupController do
           description("A group in the application")
 
           properties do
-            input_schema(:map, "Input schema")
-            locale(:string, "The configured locale for the group")
-            locale_data(:map, "Meta data about locale settings for the group")
+            name(:string, "Name of a group")
+            parent_id(:integer, "ID of a parent")
+            type(:string, "Type of a group")
+            program_type(:string, "Type of a program")
+            program_sub_type(:string, "Sub-type of a program")
+            program_mode(:string, "Mode of a program")
+            program_start_date(:date, "Starting date of a program")
+            program_target_outreach(:integer, "Target outreach for a particular program")
+            program_products_used(:string, "Products used in a program")
+            program_donor(:string, "Donor of a program")
+            batch_contact_hours_per_week(:integer, "Contact hours for a batch in a week")
+            group_input_schema(:map, "Input schema")
+            group_locale(:string, "The configured locale for the group")
+            group_locale_data(:map, "Meta data about locale settings for the group")
           end
 
           example(%{
-            input_schema: %{},
-            locale: "hi",
-            locale_data: %{
+            name: "Abhinav Singh",
+            parent_id: 1,
+            type: "program",
+            program_type: "",
+            program_sub_type: "",
+            program_mode: "Offline",
+            program_start_date: 02 / 02 / 2020,
```

Comment:
Update it to string I think?
```suggestion
            program_start_date: "02/02/2020",
```

Also, is this the right format? Or are you getting `Y-m-d`?